### DISCUSSION TITLE
My account is broken!

### DISCUSSION THREAD

**Discussion by khalidsharif** [Budibase/budibase#12498] [Help]
Dears,

I wonder why I am getting this message now and not able to login !! 

{
    "message": "Account not found for tenant: deenakanoo",
    "status": 500
}

It was working perfectly the day before yesterday, and now I want to continue to develop the APP but I cannot access the dashboard at all.

![image](https://github.com/Budibase/budibase/assets/50369621/697deb96-d36f-4ea1-95b2-3471782ea2e5)

**Comment by melohagan** [comment]
Hey @khalidsharif 

The error appears to be correct, there is no account registered as : `deenakanoo`

Are you sure the account wasn't deleted?

**Reply by khalidsharif** [reply]
I wanted to rename the account title, maybe I deleted the account by mistake ?! What I can do know to get back live ? I have to create another account ?

**Reply by melohagan** [reply]
Rename the account title... do you remember the steps you took?
